src/utils/text.ts

```diff
@@ -33,7 +37,18 @@ export function calculateEditorColFromVimScreenCol(line: string, screenCol: numb
         }
 
         if (currentCharIdx >= line.length) {
```

Comment:
```suggestion
        if (currentCharIdx >= wcswidth(line)) {
```
@ollien Here we cannot use width; what we need is character index